addon/components/detail.js

```diff
@@ -384,6 +384,17 @@ export default Component.extend(PropTypeMixin, {
     this.set('reduxStore', reduxStore)
     this.set('renderModel', reduxStore.getState().model)
     reduxStore.subscribe(this.storeUpdated.bind(this))
+
+    /**
+     * when tabName is send while calling component
+     */
+    let tabName = this.get('tabName')
+    if (tabName !== undefined) {
+      const selectedTab = this.get('cellTabs').findBy('alias', tabName)
+      if (selectedTab !== undefined) {
+        this.set('selectedTabIndex', selectedTab.id)
+      }
+    }
```

Comment:
This logic shouldn't go in the init method because a consumer may programmatically replace the entire bunsen model/view on an instance after it is initialized, at which point they'd expect the correct tab to be selected. Instead this logic should go in `didReceiveAttrs` and execute anytime the value changes.